backend/src/todo.txt

```diff
@@ -10,14 +10,15 @@ if type checking - saves memory X
 fix async (use yeild) V
```

Comment:
TODOs in python can be set in a comment like this:
`# TODO: text`
and then shift + shift and search todo etc..